_posts/2023-10-08-amazon_bedrock_part1.markdown

```diff
@@ -0,0 +1,89 @@
+
+This blog post comes in two exciting parts. In Part 1, I'll dive into the core concepts and terminology, taking apart the inner mechanisms of Amazon Bedrock. Then, brace yourself for Part 2, where I'll walk you through crafting a simple yet captivating storytelling bot with Amazon Bedrock. If you're already well-versed in the basics, feel free to jump ahead to Part 2 and embark on your creative journey! 
+
+
+## Let's Begin With basics
+
+If you share my approach of beginning with documents, you probably have already visited the "Amazon Bedrock" homepage.
+
+![BedRock Description](/img/bedrock/bedrock_home_page.png)
+
+ On this page, three significant terms stand out: "Easiest way", "Scalable", and "Foundation Models". I am convinced that these attributes and characteristics are what set Amazon Bedrock apart from any other alternatives.
+
+Ever wondered why AWS calls this the "easiest" and a "Scalable" option? Well, if you're an expert in generative AI (GAI), you've likely tasted the complexity of setting up and managing the nuts and bolts needed for a generative AI app. It's like solving a puzzle with pieces like:
+
+- Picking the right computer power.
+- Network Configurations.
+- Ensuring model and data safety.
+- Monitoring and Keeping an eye on the infrastructure for a reliable app.
+- Data security
+
+So, how does Amazon Bedrock make this complex stuff easy? Picture this: Bedrock often serves up pre-configured resources, custom-made for GAI tasks. These ready-to-go setups come with all the software bits and pieces you need, like libraries, dependencies and tools, already installed. Plus, Bedrock integrates with AWS Managed services like Amazon S3, Amazon CloudWatch, and AWS Lambda, making tough tasks like configuring data storage, authentication, and monitoring a walk in the park for your GAI apps.
+
+Here's the cool part: Amazon Bedrock is "Serverless." That means it can automatically grow or shrink resources as your app's popularity ebbs and flows. So, when traffic goes up, Bedrock scales up your resources, ensuring peak performance without breaking the bank.
+
+If I've got you excited about the ease and efficiency of GAI apps with Amazon Bedrock, it's time to dive into the world of "Foundation Models".
+
+## Uncover The Magic Of Foundation Models (FMs)
+
+FMs are like super-smart, giant neural networks trained on massive piles of data. Instead of reinventing the AI models every time from scratch, we use FMs as a launchpad to create our own customised models in a faster and cost-effective approach. These FMs are like all-in-one champs; they can multiple tasks with high accuracy, like generating image or text from a simple input prompt, answering tricky questions, and even solving math puzzles. 
```

Comment:
` they can multiple tasks` should this be like  'they can execute multiple tasks`?